docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -98,6 +98,7 @@ export type Manifest = {
   launchAsset: Asset;
   assets: Asset[];
   metadata: { [key: string]: string};
+  extra: { [key: string]: any};
```

Comment:
would it make sense to enforce that values are also key-string'd objects?

Reply:
The values really can be anything JSON-compatible right now. I believe we have objects that are nested a few levels deep. There might be some arrays too.